### PR TITLE
Re-writing the intro paragraph of §5.1

### PR DESCRIPTION
**Pull request by iherman**
The introductory paragraph in [§5.1](https://w3c.github.io/did-core/#identifiers) says:

> It is necessary to learn the identifiers used within a DID document to refer to the DID subject and the DID controller, as well as any alternate identifiers that might also be used to reference the DID subject. The following section elaborates on the mechanisms the DID document data model provides for so doing.

I was not fully sure  what this paragraph really tries to say and, in any case, "necessary to learn" is not an acceptable style for a specification. Also, the generic statement on identifiers is not accurate: identifiers in a DID document are also used to identify verification methods or services.

The PR proposes an alternate formulation based on my understanding (which may be entirely wrong!).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/683.html" title="Last updated on Feb 21, 2021, 9:09 PM UTC (3f51dfa)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/683/69f7eff...3f51dfa.html" title="Last updated on Feb 21, 2021, 9:09 PM UTC (3f51dfa)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1540,11 +1540,8 @@ <h3>Service properties</h3>
       <h2>Identifiers</h2>
 
       <p>
-It is necessary to learn the identifiers used within a <a>DID document</a> to
-refer to the <a>DID subject</a> and the <a>DID controller</a>, as well as any
-alternate identifiers that might also be used to reference the <a>DID subject</a>.
-The following section elaborates on the mechanisms the <a>DID document</a> data
-model provides for so doing.
+
+        A <a>DID document</a> must include identifiers referring to the <a>DID subject</a>, as well as to the <a>DID controller</a>. While the primary mechanism is to use the <a href="#did-syntax">DID Syntax</a>, the <a>DID document</a> may also include alternate identifiers to reference the <a>DID subject</a>. The following section elaborates on the mechanisms the <a>DID document</a> data model provides for so doing.
```
Comment (msporny) [inline index.html:1544]: Wrap to 80 characters, no indents, please. Long lines make diff-reading /very/ difficult and also complicate suggesting edits in Github.

**Review by msporny — CHANGES_REQUESTED** [review]
Wrap to 80 characters, no indents, please. Long lines make diff-reading /very/ difficult and also complicate suggesting edits in Github.

File: index.html (hunk 1 above)
Comment (brentzundel) [inline index.html:1544]:
```suggestion
It is necessary for <a>DID documents</a> t/o include identifiers referring to
the <a>DID subject</a>, as well as to the <a>DID controller</a>. While the
primary mechanism is to use the <a href="#did-syntax">DID Syntax</a>, the
<a>DID document</a> could also include alternate identifiers to reference the
<a>DID subject</a>. The following section elaborates on the mechanisms the
<a>DID document</a> data model provides for so doing.
```

Comment (brentzundel) [reply index.html:1544]: suggestion fixes line length and removes language that could be considered normative.

File: index.html [hunk 2]
```diff
@@ -1540,11 +1540,13 @@ <h3>Service properties</h3>
...
-alternate identifiers that might also be used to reference the <a>DID subject</a>.
-The following section elaborates on the mechanisms the <a>DID document</a> data
-model provides for so doing.
+
+It is necessary for <a>DID documents</a> t/o include identifiers referring to
+the <a>DID subject</a>, as well as to the <a>DID controller</a>. While the
+primary mechanism is to use the <a href="#did-syntax">DID Syntax</a>, the
+<a>DID document</a> could also include alternate identifiers to reference the
+<a>DID subject</a>. The following section elaborates on the mechanisms the
+<a>DID document</a> data model provides for so doing.
```
Comment (rhiaro) [inline index.html:1549]:
```suggestion
This section describes the mechanisms by which <a>DID documents</a>
include identifiers for <a>DID subjects</a> and <a>DID controllers</a>.
```

**Review by rhiaro — APPROVED** [review]
It seemed a bit long winded, and the part about DID syntax was hard to follow, and in attempting to trim it down my suggested edit replaces the whole thing...

File: index.html [hunk 3]
```diff
@@ -1540,11 +1540,13 @@ <h3>Service properties</h3>
       <h2>Identifiers</h2>
 
       <p>
-It is necessary to learn the identifiers used within a <a>DID document</a> to
-refer to the <a>DID subject</a> and the <a>DID controller</a>, as well as any
-alternate identifiers that might also be used to reference the <a>DID subject</a>.
-The following section elaborates on the mechanisms the <a>DID document</a> data
-model provides for so doing.
+
+It is necessary for <a>DID documents</a> t/o include identifiers referring to
```
[the remark this answers is not in the document]
Comment (msporny) [reply index.html]: Used the alternate wording.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.
